app/views/posts/show.html.erb

```diff
@@ -1 +1,27 @@
-<h1>This is the post by id file for testing</h1>
\ No newline at end of file
+<div class="container">
+  <div class="card mb-3" style="max-width: 800px;">
+    <div class="card-body text-success text-bg-light p-3">
+      <div class="d-flex justify-content-between">
+        <div>
+          <h4><span class="card-title">Post <%= @post_id %></span> <span>by <%= @post.author.name %></span></h4>
+        </div>
+        <div>
+          <span class="card-text">Comments: <%= @post.comments_counter%></span>
+          <span class="card-text">Likes: <%= @post.likes_counter%></span>
+        </div>
+      </div>
+      <p class="card-text"><%= @post.text %></p>
+    </div>
+  </div>
+  <% if @recent_comments.any? %>
```

Comment:
- [ ] On the posts show page, you are still showing the recent comments just like the previous reviewer highlighted. Please ensure to show all the comments here 👍 
From the attached screenshot below, you can see that the comment count is 6, but you are only displaying 4

![image](https://github.com/Zafron047/Blog_app_by_Rails/assets/46839548/dbd25ea2-ba53-4a9d-a1f0-62ff8a5c42fe)

Reply:
Thanks a lot for the walk through and debugging. Now it is showing only the relevant comments.

![all_comments_showjpg](https://github.com/Zafron047/Blog_app_by_Rails/assets/117600135/495edc2a-bbef-4aa8-950f-b5c292efa89d)